src/components/accessories/tabs/types.ts

```diff
@@ -0,0 +1,9 @@
+export interface IProps {
+  header: ITabConfig;
+  content: ITabConfig;
+}
+
+export interface ITabConfig {
+  mainClass: string;
+  items: Array<object>;
```

Comment:
The same thing here. Instead of using `object`, try to use `Record<number, string>` 😉 .